tests/distributions/test_transformed_distribution.py

```diff
@@ -78,15 +76,15 @@ def make_lognormal(kwargs):
     },
     {
         'example': {
-            'mu': [[1.4, 0.4, 0.4], [1.4, 0.4, 0.6]],
-            'sigma': [[2.6, 0.5, 0.5], [2.6, 0.5, 0.5]],
+            'mu': [1.4, 0.4, 0.4],
```

Comment:
Removed the repeated params, so that the `analytic_mean/var()` methods return the same shape as that of a single sample.